### PR TITLE
added a java code

### PR DESCRIPTION
**Pull request by supriyaP19**

### CODE REVIEW

**Comment by srsandy** [comment]
@sup19-19 Thanks for the contribution!
Can you pls add the test cases?

**Comment by sup19-19** [comment]
Sure. I will add it by tomorrow

File: Practice Problems/Maths/readme.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Max Points on a line
+This is a very nice question which doesn't only contain the concept of slopes and straight lines, but also contains some small but tricky mathematical concepts to be taken care of which normally is not visible.
```
Comment (srsandy) [inline Practice Problems/Maths/readme.md]: Add the test cases here with minimal formatting.
Thanks.

**Comment by srsandy** [comment]
No changes made. Closing this PR.

**Pull request closed without merging**